core/types/qi_tx.go

```diff
@@ -27,6 +27,7 @@ type WireQiTx struct {
 	TxIn      TxIns
 	TxOut     TxOuts
 	Signature []byte
+	Data      []byte
```

Comment:
I thought you were going to delete this?

Reply:
oh its a noop

Reply:
i had initially changed it then realized its not used, i can remove, or we can remove this later